client/my-sites/customer-home/style.scss

```diff
@@ -69,6 +73,10 @@
 	}
 }
 
+.customer-home__main .card {
+	border-radius: 2px;
```

Comment:
The Figma is using 3px for these corners. Might result in a lint warning, but we are intentionally breaking away from the standard Calypso style so 🤷 

```suggestion
	border-radius: 3px;
```

Reply:
> I don't know what we should do with the notch when "Classic Bright" is selected though, because the border is going to run right through the notch 🤔

Good eye @p-jackson. I switched to that theme to check it out. I think it's okay if we discover issues with other wpcom themes. We should log those issues, but I don't think letting them detract from our primary focus of My Home work. Instead of a side border there, should we just color that little notch white on my home only?

Perhaps we can use this as a push to change the rest of the calypso page bgs to white _maniacal laugh_

![Screen Shot 2021-06-25 at 1 14 55 PM](https://user-images.githubusercontent.com/214813/123461813-8a3e4400-d5b7-11eb-9823-98caf45a0a53.png)